client/stylesheets/shared/_embed.scss

```diff
@@ -6,12 +6,16 @@
 		overflow-x: initial !important;
 	}
 
+	#wpbody-content {
+		padding-top: 0;
+	}
+
 	#wpbody-content .notice {
 		margin-top: 15px;
 	}
 
 	.wrap {
-		padding: 20px 20px 0;
+		padding: $header-height 20px 0;
```

Comment:
One issue with applying this to the wrap is that store alerts will overlap with the header:

<img width="851" alt="Screen Shot 2020-06-18 at 3 29 33 PM" src="https://user-images.githubusercontent.com/10561050/85020557-2e0d3c80-b179-11ea-99fa-a8740313528d.png">